packages/edit-site/src/components/sidebar/color-panel.js

```diff
@@ -7,12 +7,11 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import {
-	BACKGROUND_COLOR,
-	LINK_COLOR,
-	TEXT_COLOR,
-	GRADIENT_COLOR,
-} from '../editor/utils';
+import { LINK_COLOR } from '../editor/utils';
+
+const BACKGROUND_COLOR = 'background-color';
```

Comment:
I'd rather keep these are they are and we can discuss how to solve it in the other PR (I'm still unconvinced). This PR is so great that it merits landing asap.